engine/common/provider/engine.go

```diff
@@ -189,7 +189,7 @@ func (e *Engine) onEntityRequest(request *internal.EntityRequest) error {
 		Strs("entity_ids", flow.IdentifierList(request.EntityIds).Strings()).
 		Logger()
 
-	lg.Debug().
+	lg.Info().
```

Comment:
The provider engine is only used by:
1. [collection engine returning collections to execution nodes](https://github.com/onflow/flow-go/blob/leo/info-log-provider-engine/cmd/collection/main.go#L395)
2. [execution nodes returning receipts to consensus nodes](https://github.com/onflow/flow-go/blob/master/cmd/execution_builder.go#L911)

The first log is very useful in debugging execution halt issue, where we need to distinguish between whether execution halt is due to execution node not sending collection requests or collection node not responding collection responses. If this log is debug level, then when execution is halted, we no longer have the log to see check whether the provider engine received the request.

The problem for changing this is it might increase the amount of logs, but this is ok, it's not too many logs, I have tested, it only add about 1-2 logs per second for collection node.
And for execution node, since consensus only request receipts when falling behind, this log should be rare.